DailyCodingProblem/Program.cs

```diff
@@ -153,5 +155,27 @@ static void Problem4()
             
             Console.WriteLine($"There is no possitive integer under or equal to {int.MaxValue} that is not a part of the array.");
         }
+
+        /// <summary>
+        /// cons(a, b) constructs a pair, and car(pair) and cdr(pair) returns the first and last element of that pair. For example, car(cons(3, 4)) returns 3, 
+        /// and cdr(cons(3, 4)) returns 4.
```

Comment:
This is a very confusing summary and the functions names don't even match the ones that you put in your code.